src/FcmClient.php

```diff
@@ -81,9 +81,13 @@ public function send(Request $request): array
         $url = $request->getUrl();
 
         // Create and send the request.
-        $response = $client->post($url, [
-            'json' => $request->getBody()
-        ]);
+        try {
+            $response = $client->post($url, [
```

Comment:
Should we also check the HTTP Status? I have a snippet like 

```
        if ($response->getStatusCode() === 400) {
            throw new \Exception("Firebase returned 400");
        } elseif ($response->getStatusCode() === 401) {
            throw new \Exception("Firebase authentication error");
        } else {
            return $response->getStatusCode() === 200;
        }
```

Reply:
That'd be useful, would you please add that as a subsequent PR? @Philipp91